### ISSUE TITLE
XML conversion of attributes still with old prefix '+' instead of '+@'

### ISSUE THREAD

**Issue by Kopfbremse** [mikefarah/yq#1428]
**Version of yq**: 4.30.1
**Operating system**: linux
**Installed via**: binary release

**Describe the bug**
When converting XML attributes, the old prefix '+' is still used instead of '+@'.

**Input XML**
```xml
<?xml version=1.0 encoding=UTF-8?>
<multiversum madness="true" />
```

**Command**
```bash
--input-format xml --output-format json```

**Output**
```json
{
  "+p_xml": "version=1.0 encoding=UTF-8",
  "multiversum": {
    "+madness": "true"
  }
}
```

**Comment by mikefarah** [comment]
doh! 🤦🏼  looks like I remembered to update the docs but not the code and tests. Will fix asap

**Comment by mikefarah** [comment]
Fixed in 4.30.2 - thanks for raising!
